README.md

```diff
@@ -54,23 +52,41 @@ Then simply import the frameworks where they are used:
 Before identifying the currently logged in user, initialize the SDK with your App ID and secret:
 
 ```swift
-    Streem.initialize(appId: "APP_ID", appSecret: "APP_SECRET")
+    Streem.initialize(delegate: self, appId: "APP_ID", appSecret: "APP_SECRET")
 ```
 
-If you are using CallKit, you should also initialize `StreemCalls` as well, so that you can make and receive phone calls:
+If you are using CallKit, you should also initialize `StreemCalls` , so that you can make and receive phone calls:
```

Comment:
We can move this initialization into the SDK. Do we need to add a `useCallKit: Bool` parameter to `Streem.initialize()`?